governance-exercises/DAOMasons-interpretation-exercise-1.md

```diff
@@ -0,0 +1,73 @@
+
+#### Situation A
+
+> Aligned Delegates are not being compensated due to a clear typo elsewhere in the Governance Scope.
+
+I would edit the typo to fix the compensation, and post on the forum that I've done so along with my reasoning and a reference to [Governance Scope 12](https://mips.makerdao.com/mips/details/MIP113#12-1-bootstrapping-governance-votes).
+
+#### Situation B
+
+> A large token holder wants to add logic to the Stability Scope scope more quickly than is possible using the monthly governance cycle. The author claims this change is required to prevent the DAO losing significant value.
+
+This is not remedying an unintended consequence or mistake, so I would not make the edit myself.
+
+This doesn't sound like an emergency, so I would attempt to judge if the situation is "[Urgent](https://mips.makerdao.com/mips/details/MIP24#MIP24c1)"
+
+I would lean on the judgement of Stability Scope facilitators here and ask if they would explicitly deem it Urgent as defined in [MIP24c3: Emergency Response Procedure](https://mips.makerdao.com/mips/details/MIP24#MIP24c3).  
+If I determine it's urgent, I'd create a poll as quickly as makes sense, and follow that with an executive. I would try to keep things in line with the standard weekly rhythms, sensing governance burden and amount of context required for voters. I think minimizing the governance load that occurs outside of the monthly governance cycle is important, but also sometimes it needs to happen.
+
+If it's not explicitly urgent but seems like a timely opportunity to claim value for the DAO, I'd still consider having an early executive vote if that week's governance burden seems like it could handle another vote.
+
+#### Situation C
+
+> An anonymous account wants to add logic to the Stability Scope scope more quickly than is possible using the monthly governance cycle. The author claims this change is required to prevent the DAO losing significant value.
+
+Same answer as above. I don't think I should treat a large token holder differently from an anon if the business case seems legitimate.
+
+If this starts happening like spam or trolling, I'd have to have some way to screen the messages before I involve other facilitators.
+
+#### Situation D
+
+> As a consequence of a significant reduction in Aligned Delegate pay, all Aligned Delegates step down from the position.
+
+If the conversation leading up to the change included acknowledgement of this risk, perhaps coupled with the intention of replacing all the AD's with others who would accept lower pay... then perhaps this is just a bumpy road toward an intended outcome.
+
+It seems more likely that this was an unintended consequence and would consider this as Urgent or an Emergency. I'd create a poll immediately to see if token holders support reversing the reduction given the reality that we currently have no AD's.
```

Comment:
Good. Keep in mind that the AD's resigning doesn't mean they lose their voting power automatically. Some consideration should be given to the fact they ADs will likely vote in favour of the comp returning. Either this should be explicitly noted, or the ADs vote-weight should be explicitly excluded from such a poll.